modules/claims_api/app/sidekiq/claims_api/poa_vbms_upload_job.rb

```diff
@@ -3,12 +3,22 @@
 require 'sidekiq'
 require 'claims_api/vbms_uploader'
 require 'claims_api/poa_vbms_sidekiq'
+require 'claims_api/claim_logger'
 
 module ClaimsApi
   class PoaVBMSUploadJob
     include Sidekiq::Job
     include ClaimsApi::PoaVbmsSidekiq
 
+    sidekiq_retries_exhausted do |message|
+      ClaimsApi::Logger.log(
+        'claims_api_retries_exhausted',
+        poa_id: message['args'].first,
+        detail: "Job retries exhausted for #{message['class']}",
+        error: message['error_message']
+      )
+    end
+
```

Comment:
why did we add this one here?

Reply:
@FonzMP  total oversight on my part, just missed it I will get that cleaned up.